quicktests/overlaying/tests/animations/animate_line.js

```diff
@@ -1,8 +1,8 @@
 
 function makeData() {
   "use strict";
-  var data = [{x: new Date("2014-04-01 00:24"), y: 4},
-              {x: new Date("2014-08-29 00:24"), y: 6}];
+  var data = [{x: new Date("4/1/2014 00:24"), y: 4},
+              {x: new Date("8/29/2014 00:24"), y: 6}];
```

Comment:
Could we use the [ISO-8601](https://en.wikipedia.org/wiki/ISO_8601) standard instead? Just for the sake of eliminating confusion.